.drone.star

```diff
@@ -2648,22 +2648,37 @@ def getPipelineNames(pipelines = []):
         names.append(pipeline["name"])
     return names
 
-def calculateTestsToRunBasedOnFilesChanged(ctx):
-    return [
-        {
-            "name": "calculate-diff",
-            "image": OC_CI_NODEJS,
-            "commands": [
-                "bash -x tests/drone/getFilesChanged.sh",
-                "ls -la",
+def skipIfUnchanged(ctx, type):
+    skip_step = {
```

Comment:
My hope would be that we can get rid of most of the `[pr-title-tags]` thanks to the skipping logic. `[full-ci]` should probably still be kept around independently from it (as in **takes no effect if you only change sth in, e.g., the changelog**) since its function is to run _all the acceptance pipelines to the end, even if one or more of them finish with a non-zero exit code_ (default behavior is stopping the whole CI run if a single acceptance pipeline fails at some point). This is mostly about debugging larger features or testing backend updates